packages/spotlight/src/container.js

```diff
@@ -638,6 +640,12 @@ function getContainerDefaultElement (containerId) {
 	}
 
 	defaultElementSelector = coerceArray(defaultElementSelector);
+
+	// If a preferred enterTo has been provided, we will favor it by making it first in search array
+	if (preferredEnterTo && typeof preferredEnterTo === 'string' && preferredEnterTo !== 'default-element') {
+		defaultElementSelector.unshift(preferredEnterTo);
```

Comment:
If only `default-element` and `last-focus` are allowed, should we just force `last-focus` here for simplicity?  If other values are allowed, should we update the docs above?